aws-java-workshop/src/main/java/com/emc/vipr/s3/sample/_06_ReadObjectWithMetadata.java

```diff
@@ -15,6 +15,9 @@
 package com.emc.vipr.s3.sample;
 
 import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
```

Comment:
As before...The Factory class has some other ways of instantiating the client.
It's currently commented out, but can be uncommented for demonstrations.